### PR TITLE
added AWSCLI support with image

### PR DESCRIPTION
**Pull request by njtran**

### CODE REVIEW

File: debug/Dockerfile [hunk 1]
```diff
@@ -16,11 +16,21 @@ RUN apk add --no-cache \
 RUN mkdir -p -m 0600 ~/.ssh && ssh-keyscan github.com >> ~/.ssh/known_hosts
 
 # Install https://github.com/ellistarn/home
-RUN --mount=type=ssh sh -c \
-    "$(wget https://raw.githubusercontent.com/ellistarn/home/main/bin/install-home.sh -O -)"
+#RUN --mount=type=ssh sh -c \
```
Comment (ellistarn) [inline debug/Dockerfile]: Leave this guy in here.

File: debug/Dockerfile [hunk 2]
```diff
@@ -16,11 +16,21 @@ RUN apk add --no-cache \
...
+#   "$(wget https://raw.githubusercontent.com/ellistarn/home/main/bin/install-home.sh -O -)"
 
 RUN --mount=type=ssh sh -c \
     "$(wget https://raw.githubusercontent.com/ellistarn/home/main/bin/install-dependencies.sh -O -)"
 
 # Set default shell to Zsh
 RUN sed -i 's|root:x:0:0:root:/root:/bin/ash|root:x:0:0:root:/root:/bin/zsh|' /etc/passwd
+
+RUN apk add --no-cache \
```
Comment (ellistarn) [inline debug/Dockerfile:29]: Nice! Can you add a comment above:
\# Install AWS CLI

File: debug/Dockerfile [hunk 3]
```diff
@@ -16,11 +16,21 @@ RUN apk add --no-cache \
...
+        python3 \
+        py3-pip \
+    && pip3 install --upgrade pip \
+    && pip3 install \
+        awscli \
+    && rm -rf /var/cache/apk/*
```
Comment (ellistarn) [inline debug/Dockerfile:35]: What's this guy for?

File: debug/Dockerfile (hunk 1 above)
Comment (njtran) [reply debug/Dockerfile]: my bad, broke for me, but i'll keep it in there.

File: debug/Dockerfile (hunk 2 above)
Comment (njtran) [reply debug/Dockerfile:29]: sure thing

File: debug/Dockerfile (hunk 3 above)
Comment (njtran) [reply debug/Dockerfile:35]: To clear the cache. Every recommended AWSCLI install for alpine recommends you do this.

Comment (ellistarn) [reply debug/Dockerfile:35]: 🥉

File: debug/Dockerfile [hunk 4]
```diff
@@ -24,3 +24,14 @@ RUN --mount=type=ssh sh -c \
...
+# Install AWS CLI
+RUN apk add --no-cache \
+        python3 \
+        py3-pip \
+    && pip3 install --upgrade pip \
+    && pip3 install \
+        awscli \
+    && rm -rf /var/cache/apk/*
+
+RUN aws --version
```
Comment (ellistarn) [inline debug/Dockerfile]: Leave this guy out as it's for your own debugging purposes.

Comment (njtran) [reply debug/Dockerfile]: got it

**Comment by ellistarn** [comment]
Thanks for the contribution! This is a great addition!

**Pull request merged**